sessionctx/variable/session.go

```diff
@@ -149,6 +149,9 @@ type SessionVars struct {
 	// GlobalAccessor is used to set and get global variables.
 	GlobalVarsAccessor GlobalVarAccessor
 
+	// LastFoundRows is the last found rows of previous query statement
```

Comment:
@ueizhou In that case, considering to add a comment for clarifying this. Without any pre-knowledge, I will assume `LastRows` is an array.